packages/theme/src/components/index.ts

```diff
@@ -1,3 +1,44 @@
+import Accordion from "./accordion"
```

Comment:
Any reason for updating the export structure here? I'd rather keep it as is.

Reply:
A bulk import (`import * from`) was used to import all the components in `packages/theme/src/index.ts`
This resulted in `typeof import("theme/src/components")` being emitted in the type definition for the `Theme`.

I agree that this adds some overhead to exporting components. I am open to suggestions on how we can fix the underlying issue and keep it as simple as possible. I didn't manage to come up with anything else than this proposal, maybe someone with more TypeScript experience could weigh in?